pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/create-an-ami/index.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: layout.pug
+navigationTitle: Create an AMI
+title: Create an AMI
+menuWeight: 9
+excerpt: Create an AMI using the DKP image builder
+enterprise: false
+---
+
+## Create an AMI to be used in an air-gapped cluster
+
+Using the [Konvoy Image Builder](../../../../image-builder), you can build an AMI without requiring access to the internet by providing an additional `--override` flag.
+
+1.  Create the directories where you will place the air-gapped bundles:
+
+    ```bash
+    mkdir artifacts
+    mkdir artifacts/images
+    ```
+
+1.  Define an environment variable for the Kubernetes version that corresponds with Konvoy release you are installing. You can find the correct Kubernetes version by checking the release notes for the release you are installing.
+
+    ```bash
+    export VERSION=1.22.8
+    ```
+
+1.  Set an environment variable for the AMI's OS you are will be using.
+    The OS packages bundles will contain the RPMs for Containerd, Kubernetes and all of their dependencies required to install these packages without access to any external RPM repositories.
+    The available options are:
+
+    * `centos_7_x86_64`
+    * `centos_7_x86_64_fips`
+    * `redhat_7_x86_64`
+    * `redhat_7_x86_64_fips`
+    * `redhat_8_x86_64`
+    * `redhat_8_x86_64_fips`
+
+    ```bash
+    export BUNDLE_OS=centos_7_x86_64
+    ```
+
+1.  Download the OS packages bundle:
+
+    ```bash
+    curl --output artifacts/"$VERSION"_"$BUNDLE_OS".tar.gz -O https://downloads.d2iq.com/dkp/airgapped/os-packages/"$VERSION"_"$BUNDLE_OS".tar.gz
+    ```
+
+1.  Download the Kubernetes images bundle. This bundle includes the necessary images for `kubeadm` to bootstrap a Kubernete `Node`.
```

Comment:
>Kubernete

Kubernetes 👍

Reply:
Found this in one place in the doc and changed it and then...

I'll update.